gulpfile.js

```diff
@@ -69,7 +69,7 @@ gulp.task('fixup-swagger', function () {
   autorest.fixupRawSwagger('./swagger/bifrost.swagger.before.json', './swagger/bifrost.swagger.json');
 });
 
-gulp.task('autorest', ['clean-autorest', 'fixup-swagger'], function () {
```

Comment:
Ah, sure. That behavior is because of the `fixup-swagger` task. Basically, I found the changes we needed to make (so far anyway) to the bifrost swagger document could be automated, so that's what it does.

What sort of changes are you making? If they're just the default required ones (adding operation IDs, removing empty paths, one other thing I don't remember ;) ) you can drop the swagger from bifrost in at the *.before.* file name and everything will just work.

If you need to do more/different than that ping me on slack/email and we'll work out the process.

Sorry about the confusion, I should have been louder about this change.